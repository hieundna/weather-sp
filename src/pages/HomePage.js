import '../styles/HomePage.scss';
import Summary from '../components/Summary/Summary';
import ChartWeather from '../components/ChartWeather/ChartWeather';
import { AppProvider } from '../contexts/context'

function HomePage() {
    return (
        <AppProvider>
            <div data-testid='container' className="container">
                <Summary />
                <ChartWeather />
            </div>
        </AppProvider>
    );
}

export default HomePage;
