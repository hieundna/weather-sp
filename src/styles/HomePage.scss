.container {
    height: 100vh;
    width: 100%;
    color: #fff;
    background-image: linear-gradient(180deg, rgba(50, 94, 189, 0.913603), rgba(25, 107, 138, 0.779149) 60%, rgba(0, 172, 255, 0.319765));
}