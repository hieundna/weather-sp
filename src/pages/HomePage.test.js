import React from 'react';
import HomePage from './HomePage.js';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const wrapper = shallow(<HomePage />);
describe('HomePage Component', () => {
    test("expect to render HomePage component", () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    })
    test('renders without error', () => {
        const homePageComponent = wrapper.find(".container");
        expect(homePageComponent.length).toBe(1);
    });

})