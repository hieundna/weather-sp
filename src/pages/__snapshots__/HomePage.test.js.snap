// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component expect to render HomePage component 1`] = `
<AppProvider>
  <div
    className="container"
    data-testid="container"
  >
    <Summary />
    <ChartWeather />
  </div>
</AppProvider>
`;
